.gitignore

```diff
@@ -12,4 +12,5 @@ dist
 *.sublime-settings
 *.db
 .nyc_output
+.history
```

Comment:
@juancarlosjr97 I would remove this snippet because I think is related with another PR, right?

Reply:
Yes, it is, I will remove it. Thank you @soyjimmysaenz